data/readme.md

```diff
@@ -0,0 +1,27 @@
+# Description:
+
+In this exercise, you are building a new data transformation off of the data within the directory [sample data](sample_data).
+
+The intended users of the transformed tables do not have much SQL experience, and it is your task to make the data easier for them to consume.
+
+For this exercise, write the transformations to allow people to see how many people are from a country by `country_name`. To demonstrate the table to the user, provide the **number of unique people from the United States and country code PR by job title** and the SQL to pull it.
+
+List out the discoveries you make along the way, and why you approached the solution the way you did.
+
+What sort of future issues do you anticipate?
+
+Describe how you will orchestrate this transformation. It is up to you if you would like to put this into code as well, or if you would rather outline the roadmap for us to talk through.
+
+You will be guaranteed an interview if you submit this exercise and will be expected to talk about the file(s) at the interview.
+
+
+# Time
+
+It's important to us that you spend at least one hour on this exercise, and NO MORE than three hours. Please set a timer for 3 hours and don't invest any additional time! This exercise is meant to be fodder for meaningful discussion with a real human, not a pass/fail or automate-able project submission.
+
+
+# Submission
+
+Please store your submission in a new private repository on github.com and share with leith@meritamerica.org. Alternatively, you may use a file-sharing service of your choice, and notify leith@meritamerica.org via email when complete. Please do not send the files as an attachment in the email as it can flag the message as spam.
```

Comment:
I put this here based on my experience that my submission never made it to you because one of our emails filtered out the zipped file due to thinking it's malicious. I would've waited forever thinking I didn't get the job unless you happened to reach out (thank you again for that!).